packages/core/src/components/menu/_menu.scss

```diff
@@ -61,10 +61,12 @@ Styleguide pt-menu
   &::before,
   .pt-icon {
     align-self: flex-start;
+    margin-top: ($menu-item-line-height - $pt-icon-size-standard) / 2;
     color: $pt-icon-color;
   }
 
   .pt-menu-item-label {
+    align-self: flex-start;
```

Comment:
seems we can just change `.pt-menu-item` to `align-items: flex-start` now, rather than overriding with `align-self` here and line 63.